homework_8_031222/src/animals/Runnable.java

```diff
@@ -0,0 +1,9 @@
+package animals;
+
+public interface Runnable {
```

Comment:
You need to segregate this interface into few of them, based on methods, Sweamable, Flyable, etc.

Reply:
Created new interfaces Swimmable, Flyable, Singable, Eatable. Classes were implemented by created interfaces.